app/uk/gov/hmrc/tai/model/YearAndMonth.scala

```diff
@@ -58,9 +74,14 @@ object YearAndMonth {
     dateTypeList.sortWith((x, y) => x.yearAndMonth.isBefore(y.yearAndMonth))
   }
 
-  private def firstClaimDate(appConfig: ApplicationConfig): YearMonth =
+  def firstClaimDate(appConfig: ApplicationConfig): YearMonth =
```

Comment:
Can this still be private?

Reply:
I tried for a while, but had to make it public so it can be called in the views with a YearMonth rather than an AppConfig in the params for formattedDate.
The only other way I could figure out that wasn't overcomplicated had an Option[YearMonth] in both and a conditional to check whether it was Some(value) or None, the None case using the firstClaimDate instead.